backend/Services/hospitalService.js

```
@@ -0,0 +1,93 @@
+      // Delete the hospital itself
+      const deletedHospital = await Hospital.findByIdAndDelete(hospitalId);
+
+      return {
+        message: "Hospital and all associated data successfully deleted",
+        deletedHospital,
+      };
+    } catch (error) {
+      return { error: error.message };
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)